workspace/Dockerfile

```diff
@@ -42,8 +38,8 @@ USER ubuntu
 RUN curl https://raw.githubusercontent.com/creationix/nvm/v0.31.0/install.sh | sh -e && \
     echo '[ -s "/home/ubuntu/.nvm/nvm.sh" ] && . "/home/ubuntu/.nvm/nvm.sh" # This loads nvm' >> /home/ubuntu/.profile
 RUN bash -l -c " \
-    nvm install 4 && \
```

Comment:
This should be a seperate PR. currently it will break some tests due to NPM on node 6 on AUFS.